recipe/validators.py

```diff
@@ -139,4 +139,4 @@ def _validate_type_scalar(self, value):
 
 RecipeSchemas(
     allowed_aggregations=list(IngredientValidator.aggregation_lookup.keys())
-).register_schemas()
+)
```

Comment:
this was unnecessary because `RecipeSchemas.__init__` already called `self.register_schemas()`.